### PR TITLE
[zh] translation for section objects & network

### PR DESCRIPTION
**Pull request by steven-my**
related: #29329
Translation for the 'objects & network' section in issue 29329.

### CODE REVIEW

**Comment by reylejano** [comment]
/retitle [zh] translation for section objects & network

File: content/zh/docs/tasks/network/validate-dual-stack.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This document shares how to validate IPv4/IPv6 dual-stack enabled Kubernetes clu
   提供可路由的 IPv4/IPv6 网络接口)
 * 一个能够支持双协议栈的
   [网络插件](/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)，
-  （如 kubenet 或 Calico）。 
+  （如 Calico，Cilium 或 Kubenet）。
```
Comment (tengqm) [inline content/zh/docs/tasks/network/validate-dual-stack.md:31]: Please fix line 27 according to the upstream English version.

Comment (steven-my) [reply content/zh/docs/tasks/network/validate-dual-stack.md:31]: 修改了27行，将Provider译成 供应商 是否正确？

Comment (tengqm) [reply content/zh/docs/tasks/network/validate-dual-stack.md:31]: Sorry, I meant line 24. The English version text in comment is incorrect

Comment (steven-my) [reply content/zh/docs/tasks/network/validate-dual-stack.md:31]: OK, reverted the line 27. Corrected line 24 in the comment.

File: content/zh/docs/tasks/network/validate-dual-stack.md [hunk 2]
```diff
@@ -21,14 +21,14 @@ This document shares how to validate IPv4/IPv6 dual-stack enabled Kubernetes clu
 
 <!--
 * Provider support for dual-stack networking (Cloud provider or otherwise must be able to provide Kubernetes nodes with routable IPv4/IPv6 network interfaces)
-* Kubenet network plugin
+* A [network plugin](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) that supports dual-stack (such as Calico, Cilium or Kubenet)
 * [Dual-stack enabled](/docs/concepts/services-networking/dual-stack/) cluster
 -->
 * 提供程序对双协议栈网络的支持 (云供应商或其他方式必须能够为 Kubernetes 节点
   提供可路由的 IPv4/IPv6 网络接口)
 * 一个能够支持双协议栈的
```
Comment (tengqm) [inline content/zh/docs/tasks/network/validate-dual-stack.md:29]:
```suggestion
* 一个能够支持[双协议栈](/zh/docs/concepts/services-networking/dual-stack/)的
```

Comment (steven-my) [reply content/zh/docs/tasks/network/validate-dual-stack.md]: 已修改

**Pull request merged**